core/src/main/java/com/github/mizool/core/rest/errorhandling/ParameterizedThrowable.java

```diff
@@ -0,0 +1,8 @@
+package com.github.mizool.core.rest.errorhandling;
+
+import java.util.Map;
+
+public interface ParameterizedThrowable
```

Comment:
I get where the name is derived from as the `GenericErrorMapper` handles `Throwable` instead of `Exception`.
I remember that it was necessary for the error mapper to be able to handle `Throwable`, so as far as I can remember, we can't change that.

However I think we shouldn't encourage developers to extend `Throwable` ever. Although this is only an interface, the name is suggestive.

Rename this to `ParameterizedException`.

Yes, technically it _could_ be applied to a `Throwable` as well and would still work. I really hope that never happens, regardless of having a good reason or not.